qa/nose_wrapper/__main__.py

```diff
@@ -6,14 +6,23 @@
 import nose.loader
 import nose.suite
 import nose.plugins.attrib
+import inspect
+
 if sys.version_info >= (3, 10) and not hasattr(collections, "Callable"):
     nose.case.collections = collections.abc
     nose.inspector.collections = collections.abc
     nose.loader.collections = collections.abc
     nose.suite.collections = collections.abc
     nose.plugins.attrib.collections = collections.abc
+if sys.version_info >= (3, 11):
```

Comment:
Total nitpick: some empty lines to separate those rather unrelated workarounds.

Reply:
Done